web/static/js/reducers/stage.js

```diff
@@ -1,5 +1,7 @@
 const stage = (state = "idea-generation", action) => {
   switch (action.type) {
+    case "SET_INITIAL_STATE":
+      return action.initialState.stage
```

Comment:
hey @caporta, great question. Because a participant can join a retro when the retro has already advanced to the `action-items` stage, `idea-generation` is really just a placeholder until the `SET_INITIAL_STATE` action gets fired with the definitive `stage` value persisted server-side.

I looked into just leaving state with no default value and got the following error:

![image](https://user-images.githubusercontent.com/14280195/27516657-66c67b80-597b-11e7-82ed-14c7e421f9ec.png)

*But!* because we need to have a default value in place on application bootstrap, we can totally just have an empty string to not be misleading. 

You cool with an empty string? Lemmeknow!

### Detour that you didn't ask to be taken on: 
Your question indirectly gets into the order of operations of application bootstrap. Right now, the OOO is `createReducer() -> retroChannel.join -> actions.setInitialState`. This is a bit much, as redux actually allows us to pass initial state directly to `createReducer`, and a better approach would be to call `retroChannel.join() -> createReducer()`, allowing us to kill the `SET_INITIAL_STATE` action entirely. The only thing keeping us from this approach is the hairy problem we discussed last week, where the retroChannel would need *access* to the store to dispatch actions to it, and we're assuming the store isn't created at that time. Let's discuss this further on Tuesday, as the elegant solution is likely staring us in the face.